### PR TITLE
Send normalized input via GRPC to server

### PR DESCRIPTION
**Pull request by pawelkowalak**
Renamed berry_client to build installable .apk. Changed GRPC to stream with
X and Y values (based on stick controller in the mobile app). Added sample
assets for controller.

### CODE REVIEW

File: berry_server/main.go [hunk 1]
```diff
@@ -2,9 +2,9 @@ package main
 
 import (
 	"fmt"
+	"io"
 	"net"
 
-	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/grpclog"
```
Comment (mateuszdyminski) [inline berry_server/main.go]: We can switch to regular logger or logrus

File: berrycli/main.go [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	log "github.com/Sirupsen/logrus"
+	pb "github.com/viru/berrybot/proto"
+)
+
+var (
+	width, height float32
+)
```
Comment (mateuszdyminski) [inline berrycli/main.go]: we could join those 2 var blocks

File: berrycli/main.go [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+				}).Info("Normalized inputs")
+				if err := stream.Send(d); err != nil {
+					log.Fatalf("%v.Send(%v) = %v", stream, d, err)
+				}
+			}
+		}
+	})
+}
+
+var conn *grpc.ClientConn
```
Comment (mateuszdyminski) [inline berrycli/main.go]: we could put those values in the top of the file

File: berrycli/main.go [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+	// Enable blending for alpha channel in PNG files.
+	glctx.Enable(gl.BLEND)
+	glctx.BlendFunc(gl.SRC_ALPHA, gl.ONE_MINUS_SRC_ALPHA)
+	images = glutil.NewImages(glctx)
+	eng = glsprite.Engine(images)
+	loadScene()
+	fps = debug.NewFPS(images)
+
+	var err error
+	conn, err = grpc.Dial("192.168.2.239:31337", grpc.WithInsecure())
```
Comment (mateuszdyminski) [inline berrycli/main.go:131]: We could try to put it in configuration, but I didn't find how : )

**Comment by mateuszdyminski** [comment]
LGTM

File: berrycli/main.go (hunk 4 above)
Comment (pawelkowalak) [inline berrycli/main.go:131]: Not sure yet how to work-around this hardcoded IP, so leaving it for now.

**Pull request merged**